src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -704,9 +704,13 @@ object DatabaseConversions extends LazyLogging {
   }
 
   implicit val blockAccountsAssociationToCheckpointRow =
-    new Conversion[List, (BlockHash, Int, Option[Instant], Option[Int], List[AccountId]), Tables.AccountsCheckpointRow] {
-      override def convert(from: (BlockHash, Int, Option[Instant], Option[Int], List[AccountId])) = {
-        val (blockHash, blockLevel, timestamp, cycle, ids) = from
+    new Conversion[
+      List,
+      (BlockHash, Int, Option[Instant], Option[Int], Option[Int], List[AccountId]),
+      Tables.AccountsCheckpointRow
+    ] {
+      override def convert(from: (BlockHash, Int, Option[Instant], Option[Int], Option[Int], List[AccountId])) = {
```

Comment:
I won't block the PR for this, but I honestly start to find it hard to make sense of so many opaque parameters, with no structure and totally common types.
I personally feel it gets hard to follow and prone to easily make mistake.

I leave it to you if there's anything to improve or not.

Reply:
I think next time when I'll be out of issues I'll improving this, I don't think it's necessary for the bugfix PR